src/assets/locales/es.json

```diff
@@ -284,6 +284,7 @@
   "commands.debug": "Alternar renderizado de depuración física.",
   "commands.vrstats": "Alternar estadísticas en realidad virtual",
   "commands.audiomode": "(experimental) Alternar audio posicional.",
+  "commands.audioNormalization": "(experimental) Change audio normalization factor.",
```

Comment:
This looks like it may have snuck in here.

Reply:
This was probably inserted automatically from the locale script we have. Probably just missed doing that in the audioNormalization PR. I think it's okay to include it in this one.

Reply:
This is also generated by `npm sync_locales.js` same as: https://github.com/mozilla/hubs/pull/3305#discussion_r521385117